CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Adds support for uploading AABs to App Distribution
```

Comment:
What is an AAB? It's not the first thing that shows up when I search it, so please spell it out here (optionally keeping the abbreviation in parenthesis).

Also end with a period please.

```suggestion
- Adds support for uploading All Aardvark Backs (AABs) to App Distribution.
```

Reply:
Done!